Automattic/woocommerce-payments
# API endpoint to fetch customer's saved payment methods.

Fixes #7322 

#### Changes proposed in this Pull Request

In this update, a new API endpoint has been implemented to retrieve the saved payment methods linked to a specific customer. This addition is designed to elevate the user experience by providing customers with a streamlined way to access and manage their stored payment options.

#### Testing instructions





* Have a **WooCommerce** customer ID (wp user id) ready with associated saved payment methods.
* Make a API GET request to the endpoint using a valid customer ID. Example: `/payments/customer/12345/payment_methods`
* Confirm a 200 OK response is received.
* Check that the returned data matches the expected saved payment methods for the customer.

-------------------


**Post merge**
